pages/api/lockerbox/delete.ts

```diff
@@ -0,0 +1,26 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+import { Role, User } from '../../../types/types'
+import { v4 as uuidv4 } from 'uuid'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      let { lockerId, label } = req.body
+
+      const locker = await prisma.lockerBox.delete({
+        where: { specificLockerBox: { label: label, lockerId: lockerId } }
```

Comment:
noob q: does this work? I was thinking it would be something like: 

```
const locker = await prisma.lockerBox.delete({
        where: {  label: label, 
        lockerId: lockerId } 
})
```

Reply:
Yeah prism makes you create an index like this for multiple where